qemu/tests/edk2_check_mor.py

```diff
@@ -0,0 +1,82 @@
+import re
+from virttest import env_process
+from virttest import error_context
+from virttest import utils_misc
+from virttest import utils_package
+from avocado.utils import process
+from avocado.utils.path import find_command
+from avocado.utils.path import CmdNotFoundError
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Verify MOR enabled in edk2 build
+
+    1. Boot guest under secure mode and check if the guest is signed
+    2. Check if secure boot is enabled inside guest
+    3. Reboot and shutdown the guest
+    4. Check MOR message after shutdown the guest
+
+    :param test: Kvm test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    def _check_signed():
+        """ Check and return if guest is signed """
+        return True if re.search(sign_keyword, sign_info) else False
+
+    package = params["package_installed"]
+    install_status = utils_package.package_install(package)
```

Comment:
Do we need to uninstall the package post-test?

Reply:
No, this package is very small and it contains test scripts, I think it can be kept. Thanks.
(Of course, we can call the function utils_package.package_remove(), but I think we can keep it here.)